test/uk/gov/hmrc/personaldetailsvalidation/endpoints/PersonalDetailsCollectionControllerSpec.scala

```diff
@@ -765,6 +765,61 @@ class PersonalDetailsCollectionControllerSpec extends UnitSpec with MockFactory
 
   }
 
+  "incorrect-details" should {
```

Comment:
Once the above comment has been addressed can you add a test for it here please.